Library/Formula/fossil.rb

```diff
@@ -28,4 +26,8 @@ def install
     system "make"
     bin.install "fossil"
   end
+
+  test do
+    system "#{bin}/fossil", "version"
```

Comment:
Any chance of a slightly more involved test (e.g. creating a fossil repo)? To be clear: this is definitely better than nothing.